src/Microsoft.AspNetCore.Server.Kestrel/Http/FrameHeaders.Generated.cs

```diff
@@ -3737,6 +3737,51 @@ protected override bool RemoveFast(string key)
+            if (((_bits & 256L) != 0)) _Via = default(StringValues);
+            if (((_bits & 512L) != 0)) _Warning = default(StringValues);
+            if (((_bits & 1024L) != 0)) _Allow = default(StringValues);
+            if (((_bits & 2048L) != 0)) _ContentLength = default(StringValues);
+            if (((_bits & 4096L) != 0)) _ContentType = default(StringValues);
+            if (((_bits & 8192L) != 0)) _ContentEncoding = default(StringValues);
+            if (((_bits & 16384L) != 0)) _ContentLanguage = default(StringValues);
+            if (((_bits & 32768L) != 0)) _ContentLocation = default(StringValues);
+            if (((_bits & 65536L) != 0)) _ContentMD5 = default(StringValues);
+            if (((_bits & 131072L) != 0)) _ContentRange = default(StringValues);
+            if (((_bits & 262144L) != 0)) _Expires = default(StringValues);
+            if (((_bits & 524288L) != 0)) _LastModified = default(StringValues);
+            if (((_bits & 1048576L) != 0)) _Accept = default(StringValues);
+            if (((_bits & 2097152L) != 0)) _AcceptCharset = default(StringValues);
+            if (((_bits & 4194304L) != 0)) _AcceptEncoding = default(StringValues);
+            if (((_bits & 8388608L) != 0)) _AcceptLanguage = default(StringValues);
+            if (((_bits & 16777216L) != 0)) _Authorization = default(StringValues);
+            if (((_bits & 33554432L) != 0)) _Cookie = default(StringValues);
+            if (((_bits & 67108864L) != 0)) _Expect = default(StringValues);
+            if (((_bits & 134217728L) != 0)) _From = default(StringValues);
+            if (((_bits & 268435456L) != 0)) _Host = default(StringValues);
+            if (((_bits & 536870912L) != 0)) _IfMatch = default(StringValues);
+            if (((_bits & 1073741824L) != 0)) _IfModifiedSince = default(StringValues);
+            if (((_bits & 2147483648L) != 0)) _IfNoneMatch = default(StringValues);
+            if (((_bits & 4294967296L) != 0)) _IfRange = default(StringValues);
+            if (((_bits & 8589934592L) != 0)) _IfUnmodifiedSince = default(StringValues);
+            if (((_bits & 17179869184L) != 0)) _MaxForwards = default(StringValues);
+            if (((_bits & 34359738368L) != 0)) _ProxyAuthorization = default(StringValues);
+            if (((_bits & 68719476736L) != 0)) _Referer = default(StringValues);
+            if (((_bits & 137438953472L) != 0)) _Range = default(StringValues);
+            if (((_bits & 274877906944L) != 0)) _TE = default(StringValues);
+            if (((_bits & 549755813888L) != 0)) _Translate = default(StringValues);
+            if (((_bits & 1099511627776L) != 0)) _UserAgent = default(StringValues);
+            if (((_bits & 2199023255552L) != 0)) _Origin = default(StringValues);
+            if (((_bits & 4398046511104L) != 0)) _AccessControlRequestMethod = default(StringValues);
+            if (((_bits & 8796093022208L) != 0)) _AccessControlRequestHeaders = default(StringValues);
```

Comment:
Could do it in one statement by putting them in a struct and doing `_headers = default(Headers)`